backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,393 @@
+showDuggaservice.php
+
+—------------------------------------------------------------------------------------------------------------------------
+TEST #1 - 100-105
+Get active users
+
+***
+Pre-req: -
+***
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = hash
+
+    }
+
+    SQL-query: SELECT active_users FROM groupdugga WHERE hash=hash
```

Comment:
this query is no longer required (im not sure if it should be removed or not.. )

Reply:
Since im also not sure i added a comment that this test is maybe not needed